src/deploymentmanager.ts

```diff
@@ -198,42 +202,53 @@ export class DeploymentManager implements IDeploymentManager {
-                    config.AzureStorageAccountKey = outputs.storageAccountKey.value;
-                    config.AzureStorageAccountName = outputs.storageAccountName.value;
-                    config.AzureStorageEndpointSuffix = storageEndpointSuffix;
-                    // If we are under the plan limit then we should have received a query key
-                    config.AzureMapsKey = outputs.azureMapsKey.value;
-                    config.CloudType = this.getCloudType(this._environment.name);
-                    config.SolutionName = answers.solutionName;
-                    config.IotHubName = outputs.iotHubHostName.value;
-                    config.SubscriptionId = outputs.subscriptionId.value;
-                    config.DeploymentId = answers.deploymentId;
-                    config.DiagnosticsEndpointUrl = answers.diagnosticsEndpointUrl;
-                    config.DockerTag = answers.dockerTag;
-                    config.DNS = outputs.agentFQDN.value;
-                    config.DocumentDBConnectionString = outputs.documentDBConnectionString.value;
-                    config.EventHubEndpoint = outputs.eventHubEndpoint.value;
-                    config.EventHubName = outputs.eventHubName.value;
-                    config.EventHubPartitions = outputs.eventHubPartitions.value.toString();
-                    config.IoTHubConnectionString = outputs.iotHubConnectionString.value;
-                    config.LoadBalancerIP = outputs.loadBalancerIp.value;
-                    config.Runtime = answers.runtime;
-                    config.SolutionType = this._solutionType;
-                    config.TLS = answers.certData;
-                    config.MessagesEventHubConnectionString = outputs.messagesEventHubConnectionString.value;
-                    config.MessagesEventHubName = outputs.messagesEventHubName.value;
-                    config.TelemetryStorgeType = outputs.telemetryStorageType.value;
-                    config.TSIDataAccessFQDN = outputs.tsiDataAccessFQDN.value;
-                    const k8sMananger: IK8sManager = new K8sManager('default', kubeConfigPath, config);
-                    deployUI.start('Setting up Kubernetes');
-                    return k8sMananger.setupAll();
+                    const aksClusterName: string = outputs.containerServiceName.value;
+                    const client = new NetworkManagementClient(this._credentials, this._subscriptionId);
+                    const aksResourGroup: string = 
+                    `MC_${outputs.resourceGroup.value}_${aksClusterName}_${resourceGroup.location}`;
+                    const moveInfo: ResourceModels.ResourcesMoveInfo = {
+                        resources: [ outputs.publicIPResourceId.value ],
+                        targetResourceGroup: `/subscriptions/${this._subscriptionId}/resourceGroups/${aksResourGroup}`
+                    };
+                    return this._client.resources.moveResources(outputs.resourceGroup.value, moveInfo)
```

Comment:
will the resource group for public IP resource still be remained?

Reply:
Added comment.